docs/source/linkintegrity.rst

```diff
@@ -0,0 +1,19 @@
+Link integrity
+==============
+
+The ``@linkintegrity`` endpoint return a list of possible reference breaches for a list of contents.
+
+You can call the ``/linkintegrity`` endpoint on site root with a ``POST`` request and a list of uids in JSON BODY:
+
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/linkintegrity_post.req
+
+The server will respond with the results:
+
+.. literalinclude:: ../../src/plone/restapi/tests/http-examples/linkintegrity_post.resp
+   :language: http
+
+The endpoint accepts only a parameter:
+
+  - ``uids``
```

Comment:
@cekk I'd like to see a test that deletes an object that contains an object that would break link integrity. To make sure Plone takes care of this.

Note to self: We have to use uids because the user might select multiple elements (in folder contents).

Reply:
added that tests